liquid-staking/src/setup/vote.rs

```diff
@@ -0,0 +1,75 @@
+use crate::basics::{
+    constants::{Epoch, Timestamp},
+    errors::{ERROR_GOVERNANCE_SC_NOT_SET, ERROR_INVALID_SC_ADDRESS},
+};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+#[type_abi]
+#[derive(
+    TopEncode, TopDecode, NestedEncode, NestedDecode, Clone, PartialEq, Eq, Debug, ManagedVecItem,
+)]
+pub struct LockedFunds<M>
+where
+    M: ManagedTypeApi,
+{
+    pub funds: EsdtTokenPayment<M>,
+    pub claim_back: Timestamp,
+}
+
+#[multiversx_sc::module]
+pub trait VoteModule {
+    #[only_owner]
+    #[endpoint]
+    fn set_governance_contract(&self, sc_address: ManagedAddress) {
+        self.governance_contract().set(sc_address);
+    }
+
+    #[only_owner]
+    #[endpoint]
+    fn set_lock_vote_period(&self, sc_address: ManagedAddress) {
+        self.governance_contract().set(sc_address);
```

Comment:
Both `set_governance_contract` and `set_lock_vote_period` set the governance contract address. Is this intentional? If not, clarify or remove redundancy.

Reply:
probably I had a naming idea and I dropped it. This endpoint shouldn't have existed